shortfin/python/shortfin_apps/sd/examples/sdxl_flags_gfx942.txt

```diff
@@ -0,0 +1,23 @@
+all
```

Comment:
Just want to appreciate that this a lot easier to maintain than how we were doing in SHARK-ModelDev :). Related question: Do we have a list of target architectures that we promise to support when we ship? Might be worth putting in some command line help info at least

Reply:
For V1.0 we are just on gfx942. I'll add in choices to the CLI

Reply:
Nice, sounds good!